jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/popup/CodeWhispererPopupManager.kt

```diff
@@ -395,6 +400,20 @@ class CodeWhispererPopupManager {
             ACTION_EDITOR_BACKSPACE,
             CodeWhispererPopupBackspaceHandler(actionManager.getActionHandler(ACTION_EDITOR_BACKSPACE), states)
         )
+
+        val actions = ComponentUtil.getClientProperty(states.requestContext.editor.component, ACTIONS_KEY) ?: return
+        actions.forEach {
+            val shortcutSet = it.shortcutSet
+            val shortcuts = shortcutSet.shortcuts
+            if (shortcuts.isEmpty() || shortcuts[0] !is KeyboardShortcut) return@forEach
+            if ((shortcuts[0] as KeyboardShortcut).firstKeyStroke.keyCode == 9) {
```

Comment:
Andrew, a naive q, where I can look up the keyCode (9 here)

Reply:
it's tab key, I will use a different implementation to make it more clear.